e2e/cypress/integration/enterprise/system_console/compliance/download_bot_compliance_export_file_spec.js

```diff
@@ -0,0 +1,125 @@
+
+const ExportFormatActiance = 'Actiance XML';
+let targetDownload;
+let fileURL;
+let pwd;
+
+describe('Compliance Export', () => {
+    let newTeam;
+    let newChannel;
+    let botId;
+    let botName;
+
+    before(() => {
+        cy.exec('PWD').then((result) => {
+            pwd = result.stdout;
+            targetDownload = path.join(pwd, 'Downloads');
+        });
+
+        cy.apiRequireLicenseForFeature('Compliance');
+
+        cy.apiUpdateConfig({
+            MessageExportSettings: {
+                ExportFormat: 'csv',
+                DownloadExportResults: true,
+            },
+            ServiceSettings: {
+                EnforceMultifactorAuthentication: false,
+                EnableBotAccountCreation: true,
+            },
+        });
+
+        cy.apiInitSetup().then(({team, channel}) => {
+            newTeam = team;
+            newChannel = channel;
+        });
+
+        //# Create a test bot
+        cy.apiCreateBot().then(({bot}) => {
+            ({user_id: botId, display_name: botName} = bot);
+            cy.apiPatchUserRoles(bot.user_id, ['system_admin', 'system_user']);
+        });
+    });
+
+    afterEach(() => {
+        deleteExportFolder();
+    });
+
+    it('MM-T1175_1 - UserType identifies that the message is posted by a bot', () => {
+        // # Post bot message
+        cy.postBOTMessage(newTeam, newChannel, botId, botName, 'This is CSV bot message');
+
+        // # Go to Compliance page and Run report
+        cy.gotoCompliancePage();
+        cy.enableComplianceExport();
+        cy.exportCompliance();
+
+        // # Download and Unzip Export file
+        downloadAndUnzipExportFile();
+
+        // * Export file should contain bot messages
+        cy.readFile(`${targetDownload}/posts.csv`).should('exist').and('have.string', `This is CSV bot message ${botName},message,bot`);
+    });
+
+    it('MM-T1175_2 - UserType identifies that the message is posted by a bot', () => {
+        // # Create token for the bot
+        cy.postBOTMessage(newTeam, newChannel, botId, botName, 'This is XML bot message');
+
+        // # Go to Compliance and enabele Run export
+        cy.gotoCompliancePage();
+        cy.enableComplianceExport(ExportFormatActiance);
+        cy.exportCompliance();
+
+        // # Download and Unzip Export File
+        downloadAndUnzipExportFile();
+
+        // * Export file should contain Delete text
+        getXMLFile().then((result) => {
+            cy.readFile(result.stdout).should('exist').
+                and('have.string', `This is XML bot message ${botName}`).
+                and('have.string', '<UserType>bot</UserType>');
+        });
+    });
+});
+
+const getXMLFile = () => {
+    // Finding xml file location
+    return cy.exec(`find ${targetDownload} -name '*.xml'`);
+};
+
+function deleteExportFolder() {
+    // # Delete local download folder
+    cy.exec(`rm -rf ${targetDownload}`);
+}
+
+function downloadAndUnzipExportFile() {
```

Comment:
These and above two functions were used in other spec. With that, please move to a common `helpers.js` and reuse.